### ISSUE TITLE
pre_redirect add_action not working

### ISSUE THREAD

**Issue by ankur0890** [MatthewC/yourls-password-protection#7]
Hi
I installed your plugin into the system. Everything works great. I am able to save a password against the URL in the options in database . But however, your  `pre_redirect` is not working. Function defined in the `yourls_add_action( 'pre_redirect', 'warning_redirection' );` is not getting called whenever the shortened link gets open. Any help or idea why this is happening. The rest of the functionality is working absolutely fine. Am I missing something over here??

**Comment by ankur0890** [comment]
Ok I inspected more into the stuff. Wrong URL scenario is happening I don't know from where

So I created a short URL ->` abc.com/<my-short-url>` and set the password on it using the configuration and it got saved also. Now problem is password check is coming on `abc.com/admin/<short-url>` not on `abc.com/<short-url>`. Your redirect function is checking for passwords after` /admin ` subdomain not on the original URL. Any idea where I am missing any config???? I am not able to figure it out yet.
